src/components/UserSwitch/UserSwitch.js

```diff
@@ -14,6 +14,10 @@ const UserSwitch = ({
   useEffect(() => {
     if (auth) {
       if (!currentUserType) {
+        if("student" === window.location.pathname.slice(-7)) 

```

Comment:
instead of seeing hardcoded last chars of URL can't we handle that using react router?(see in app.js)